src/GridFS/WritableStream.php

```diff
@@ -96,7 +104,11 @@ public function __construct(CollectionWrapper $collectionWrapper, $filename, arr
 
         $this->chunkSize = $options['chunkSizeBytes'];
         $this->collectionWrapper = $collectionWrapper;
-        $this->ctx = hash_init('md5');
+        $this->disableMD5 = isset($options['disableMD5']) ? $options['disableMD5'] : false;
```

Comment:
As with Bucket's constructor, `'disableMD5' => false,` when initially appending `$options` and then this can be a basic assignment.